### PR TITLE
feat: to-be process

### LINKED ISSUE #192: To-be process
**User story**
As a user I want to map how a perfect process would look like so that we can communicate around the targets

**Describe the solution you'd like**
- A special case to the version history in future
- Based on current version

**Dependency**
#154 

----
### NOTE: 
- ~~#159 needs to be done in the API before we can start on this task.~~ Labels are now supported in the api.
---- 
# Development breakdown
image

This new feature depends on #159 

**Background**
A `to-be` process is an alternative of the same `original` process, that is mapped how someone imagines a _**perfect**_ ✨ process to be.
The user should be able to quickly change between these two alternatives and compare them together.
The `to-be` process is technically and practically a new process with its own Problems, Questions, Ideas and Risks. Though it shares the same name, labels and access-roles.

### Requirements
- [x] User can flip a toggle to switch between the `original` and the `to-be` process.
- [x] Name is shared
- [x] Access is the same as the original process
- [x] ~Labels are shared between the `original` and the `to-be` processes.~ Depends on #159
- [x] To-be processes should not be shown on the front-page. _(We need some metadata to distinguish them from regular ones...)_
- [x] Individual version history.
- [x] PQIRs are not shared between `original` and `to-be`.
- [x] Disable toggle if `to-be` is not created & you don't have access.
- [x] When duplicating a process, it should duplicate the `current`-process and not create a `To-be`-process.
- [ ] When updating a `To-be` process, it should be reflected in the front-page `last-modified`.

**Use-case**
The API handles all requests related to labels, process-name, and access to the `to-be` process it as if it was the `original` process.

Ex 1.
Client ask for labels for a `to-be` process.
The API answers with the labels for the `original`.

Ex 2
Client adds a label to a `to-be` process.
The API handles it as if it was the `original` process.

----
TODO:
- [ ] Reset Api-endpoint
- [x] Change to-be process toggle
- [x] Update the delete text and dialog design
- [x] Reset button
- [x] Add reset button dialog

### PR DESCRIPTION
**Pull request by SjoenH**
**Background**
A `to-be` process is an alternative of the same `original` process, that is mapped how someone imagines a _**perfect**_ ✨ process to be.
The user should be able to quickly change between these two alternatives and compare them together.
The `to-be` process is technically and practically a new process with its own Problems, Questions, Ideas and Risks. Though it shares the same name, labels and access-roles.

**Requirements**
- [x] User can flip a toggle to switch between the `original` and the `to-be` process.
- [x] Name is shared
- [x] Access is the same as the original process
- [x] ~Labels are shared between the `original` and the `to-be` processes.~ Labels are not yet implemented in the front-end, but should work... #159 
- [x] To-be processes should not be shown on the front-page. _(We need some metadata to distinguish them from regular ones...)_
- [x] Individual version history.
- [x] PQIRs are not shared between `original` and `to-be`.
- [x] When duplicating a process, it should always duplicate the `current`-process. (Not the `To-be`-process)

Two new requirements:
- [x] Change to-be process toggle to new design. (ref [Figma](https://www.figma.com/file/IkHwmIQrsT0iR34f5R5UnZ/VSM?node-id=2994%3A117375)) 
image
- [x] Reset button on the to-be-process
- [x] Update delete text as described in https://github.com/equinor/MAD-VSM-WEB/issues/192#issuecomment-966144168

**Use-case**
The API handles all requests related to labels, process-name, and access to the `to-be` process it as if it was the `original` process.

Ex 1.
Client ask for labels for a `to-be` process.
The API answers with the labels for the `original`.

Ex 2
Client adds a label to a `to-be` process.
The API handles it as if it was the `original` process.

closes #192

### CODE REVIEW

**Review by dansvend — APPROVED** [review]
LGTM👌

**Comment by SjoenH** [comment]
Adding another requirement to this PR:
- [ ] When duplicating a process, it should duplicate the `current`-process and not create a `To-be`-process.

**Review by dansvend — APPROVED** [review]

**Comment by SjoenH** [comment]
Two new requirements:
- [ ] Reset button on the to-be-process
- [ ] Change to-be process toggle

Added to the description.

**Comment by SjoenH** [comment]
Deleting a process now looks like this 
image

File: layouts/default.layout.module.scss [hunk 1]
```diff
@@ -20,29 +20,14 @@
...
   line-height: 24px;
   /* identical to box height, or 150% */
 
-  letter-spacing: 0.2px;
-
-  /* Interactive Primary/⚪️/Resting */
-  color: #007079;
-
-  //border-bottom: solid #DCDCDC;
+  /* Text + static icons/⚪️/Default */
```
Comment (trond-eskeland) [inline layouts/default.layout.module.scss:29]: ⚪️/

**Review by trond-eskeland — APPROVED** [review]
Looks good 👍

**Pull request merged**